SCClassLibrary/Common/Core/Object.sc

```diff
@@ -389,6 +389,7 @@ Object  {
 	dereference { ^this } // see Ref::dereference
 	reference { ^Ref.new(this) }
 	asRef { ^Ref.new(this) }
+	dereferenceOperand { ^this }
```

Comment:
Per https://github.com/supercollider/supercollider/pull/2802#issuecomment-308713690, are we leaving `dereferenceOperand` in place?